benchmarks/acou3d.cpp

```diff
@@ -147,7 +147,7 @@ void setupInitialConditions(Array<float,3>& P1, Array<float,3>& P2,
         Range(cavityFront,cavityBack)) = 0.001;
 
     // Initial pressure distribution
-    BZ_USING_NAMESPACE(blitz::tensor);
+    using namespace blitz::tensor;;
```

Comment:
Can you remove double semicolons with greb/sed?

Reply:
Good catch... And easy to do as there are 3 occurences (related to the bad use of the macro...).
Will be done in a minute.

Note that there are many other spurious ;; generated in loops from the python script. Corrected too.